python/cuda_cccl/test/test_cuda_cccl.py

```diff
@@ -0,0 +1,21 @@
+import cuda.cccl as c4l
```

Comment:
If `c4l` is an analogous naming convention to e.g., `a11y` or `k8s`, wouldn't it technically be `c2l`? ;)

Reply:
My train of thought was (C)uda.(C)(C)(C)(L) translates to `ccccl`, which I abbreviated to `c^4l` and serialized to `c4l`.

Reply:
I would advocate that we do not introduce a new internal name here. Let’s stick to just `cccl`, please.
```suggestion
from cuda import cccl
```

Reply:
Gotcha. i don't want to nitpick here, but before this convention proliferates, it's probably a good idea to check with @jrhemstad if we want to start abbreviating `CUDA CCCL` to `c4l` (or `c3l`). I do like it!

Reply:
Sorry, I posted my comment before I saw Bradley's response.

Reply:
Used `from cuda import cccl` as suggested.